data/consent-page.js

```diff
@@ -0,0 +1,66 @@
+const yo = require('yo-yo');
+
+// Build the study header
+const buildStudyTitle = studyName => {
+  const titleText = studyName || 'Participate in a new Firefox Study';
+  return yo`<h1>${titleText}</h1>`;
+};
+
+// Build the opt-in button
+const buildParticipateButton = (buttonText, buttonClasses) => {
+  const text = buttonText || `Try ${self.options.addonName}`;
+  return yo`
+  <button id="install-addon" class=${buttonClasses || 'primary'}>
```

Comment:
Separately, I think we need to have `class="${buttonClasses || 'primary}"`, ie: with quotes around the classname. Or does yoyo deal with that for us?